Improvements to DigitalOcean App Platform CLI Example (#268)
1Password/scim-examples: beta/do-app-platform-op-cli/README.md

  > **Note**
  >
  > If you don't have a DigitalOcean account, you can sign up for a free trial with starting credit: <https://try.digitalocean.com/freetrialoffer/>

> - Access to a terminal on Mac or Linux Bash, Zsh, or on Windows using Powershell

Reviewer: Coming back to this, this feels somewhat clunky. 

maybe: "Zsh or Bash on Mac or Linux, or PowerShell on Windows"?

> - Access to a terminal on Mac or Linux Bash, Zsh, or on Windows using Powershell

Author: Applying your suggestion.

```suggestion
- Access to Zsh or Bash on Mac or Linux, or PowerShell on Windows
```

## Getting started

### Step 1: Install 1Password and DigitalOcean tools

> Install the following on your Mac, Linux or Windows machine:

Reviewer: Order in style guide order (Mac, Windows, Linux)

Author: ```suggestion
Install the following on your Mac, Windows or Linux machine:
```

> - 1Password 8 for [Mac](https://1password.com/downloads/mac/) or [Linux](https://1password.com/downloads/linux/)

Author: Add Windows download link to 1Password 8
```suggestion
- 1Password 8 for [Mac](https://1password.com/downloads/mac/) or [Linux](https://1password.com/downloads/linux/) or [Windows](https://1password.com/downloads/windows/)
```

- [1Password CLI 2.9.0](https://developer.1password.com/docs/cli/get-started/#install) or later
- [`doctl`](https://docs.digitalocean.com/reference/doctl/how-to/install/#step-1-install-doctl)
  > **Note**
  >
  > **Only** [Step 1: Install doctl](https://docs.digitalocean.com/reference/doctl/how-to/install/#step-1-install-doctl) in the `doctl` installation guide is required for this step for Mac and Linux. 

> ### Step 2: Add your 1Password account and configure the DigitalOcean access.

Reviewer: Access feels weird? 

"Configure DigitalOcean credentials"?

Author: ```suggestion
### Step 2: Add your 1Password account and configure your DigitalOcean credentials.
```

Author: ```suggestion
### Step 2: Add your 1Password account and configure your DigitalOcean credentials.
```

If you haven't already done so, add your 1Password account and connect 1Password CLI to your desktop app, then configure the DigitalOcean shell plugin:

> 1. [Add your 1Password account](https://support.1password.com/add-account/) to 1Password 8 for Mac or Linux or Windows.

Reviewer: ditto, order Mac, Windows, Linux

Author: ```suggestion
1. [Add your 1Password account](https://support.1password.com/add-account/) to 1Password 8 for Mac, Windows or Linux.
```

  ```

  Using Powershell:
  ```pwsh
  op document create "pathToFile/workspace-credentials.json" --title "workspace-credentials" --vault "op-scim"
  op item edit "workspace-credentials" --vault "op-scim" base64_encoded_credentials=$([Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes($(op read "op://op-scim/workspace-credentials/workspace-credentials.json"))))
  ```

  3. Download and open the  [`workspace-settings.json`](/beta/do-app-platform-op-cli/google-workspace/workspace-settings.json)) file in a text editor and replace the values for each key:

  - `actor`: the email address for the administrator that the service account is acting on behalf of
  - `bridgeAddress`: the URL you will use for your SCIM bridge (not your 1Password account sign-in address). This is the Application URL for your Container App found on the overview page. For example: https://op-scim-bridge-example.ondigitalocean.app

  ```json
  {
      "actor":"admin@example.com",
      "bridgeAddress":"https://op-scim-bridge-example.ondigitalocean.app"
  }
  ```

  Save the file.

>   4. Configure the `workspace-settings` for passing to App Platform, just like the `workspace-credentials`, this value needs to the passed to the App Platform as a base64 value. Edit the commands to reference the location/path of the file you saved in the last step.

Reviewer: "Needs to be passed"

Author: ```suggestion
  4. Configure the `workspace-settings` for passing to App Platform, just like the `workspace-credentials`, this value is passed to the App Platform as a base64 value. Edit the commands to reference the location/path of the file you saved in the last step.
```
